engine/src/cmdsc.cpp

```diff
@@ -3416,7 +3416,8 @@ void MCReplace::exec_ctxt(MCExecContext& ctxt)
     
     
     MCAutoStringRef t_target;
-    if (!ctxt . EvalExprAsMutableStringRef(container, EE_REPLACE_BADCONTAINER, &t_target))
+    // PM-2015-05-15: [[ Bug 15352 ]] Prevent crash when setting label field contents to empty
+    if (!ctxt . EvalOptionalExprAsStringRef(container, kMCEmptyString, EE_REPLACE_BADCONTAINER, &t_target))
```

Comment:
I'm not sure I quite see how this can fix the issue... EvalOptional is keyed on whether 'container' is nil or not rather than anything else... As 'container' is the target in the expression I'm guessing EvalExprAsMutableStringRef is needed to ensure it is a mutable string which is created on which ExecReplace acts.